btc-lsp/nix/k8s-upgrade.sh

```diff
@@ -4,73 +4,110 @@ set -e
 
 THIS_DIR="$(dirname "$(realpath "$0")")"
 BUILD_DIR="$THIS_DIR/../build"
-SETUP_MODE="--keep"
+BITCOIN_NETWORK="regtest"
+SETUP_MODE="keep"
+SERVICES=""
 GITHUB_RELEASE="$(cat "$THIS_DIR/../../VERSION" | tr -d '\n')"
+K8S_CTX=`kubectl config current-context || echo ""`
```

Comment:
maybe we should use "btc-lsp" context by default as its the one defined by minikube?